src/content/en/fundamentals/performance/rail.md

```diff
@@ -132,21 +142,16 @@ To evaluate your site against RAIL metrics, use the Chrome DevTools [Timeline to
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms.</td>
-      <td data-th="User Test">User taps on an icon or button (for example, opening the nav menu, tapping Compose).</td>
+      <td data-th="User Test">User taps a button (for example, opening navigation).</td>
     </tr>
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 16ms.</td>
-      <td data-th="User Test">User drags their finger and the app's response is bound to the finger position (for example, pull to refresh, swiping a carousel).</td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Animation</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms for initial response.</td>
       <td data-th="User Test">User initiates page scroll or animation begins.</td>
     </tr>
     <tr>
       <td data-th="RAIL Step"><strong>Animation</strong></td>
-      <td data-th="Key Metric">Each frame's work (JS to paint) completes < 16ms.</td>
+      <td data-th="Key Metric">Each frame's work (from JS to paint) completes < 16ms.</td>
       <td data-th="User Test">User scrolls the page or sees an animation.</td>
     </tr>
     <tr>
```

Comment:
On line 169 - Satisfy the Response goals during the full page load process.

That's pretty intense! Do we actually expect developers to be able to accomplish that? I don't think I've heard anyone claiming this.